mathics/core/formatter.py

```diff
@@ -119,3 +151,386 @@ def add_conversion_fn(cls, module_fn_name=None) -> None:
 
     # Finally register the mapping: (Builtin-class, conversion name) -> conversion_function.
     format2fn[(conversion_type, cls)] = module_dict[module_fn_name]
+
+
+class _BoxedString(BoxElement):
+    value: str
+    box_options: dict
+    options = {
+        "System`ShowStringCharacters": "False",
+    }
+
+    @property
+    def head(self):
+        return SymbolString
+
+    def __init__(self, string: str, **options):
+        self.value = string
+        self.box_options = {
+            "System`ShowStringCharacters": SymbolFalse,
+        }
+        self.box_options.update(options)
+
+    def __repr__(self):
+        return self.value
+
+    def __str__(self):
+        return self.value
+
+    def boxes_to_text(self, **options):
```

Comment:
I wonder if we could put all out all of the `to_text()`, methods here and in other modules and create that in a  `mathics.format.text` module similar for what we have done for JSON, Asymptote, and SVG. Likewise for `MathML`, and `TeX`. 

Having all of the format-related conversions in one module allows adding another kind of backend-format module easier because we can inspect other similar modules and adapt those.  Personally, I have found it helpful that all of the asymptote routines are in one module rather than each of the classes that could potentially produce asymptote output. 

What do you think?